src/explore-education-statistics-frontend/src/modules/find-statistics/PublicationReleasePage.tsx

```diff
@@ -490,6 +490,7 @@ const PublicationReleasePage: NextPage<Props> = ({ release }) => {
       {release.content.length > 0 && (
         <Accordion
           id="content"
+          visuallyHiddenText={`${release.title} sections`}
```

Comment:
Same here, as it's in the context of a release it probably doesn't need to repeat the title here so could just use the default hidden text.